redirector.py

```diff
@@ -172,6 +171,22 @@ def get(self):
     def stats(self):
         return self.settings['stats']
 
+class APISpawnHandler(RequestHandler):
+    @gen.coroutine
+    def post(self):
+        random_host = select_host(self.stats)
+        http_client = AsyncHTTPClient()
+        request = HTTPRequest(random_host + "/api/spawn/", method="POST", body="")
+        response = yield http_client.fetch(request)
+        data = json.loads(response.body.decode("utf-8"))
```

Comment:
Make sure to wrap this in a `try` block so we can deal with failure. I'm not sure whether we pass the error response back on to the user or what.